app/assets/javascripts/shortcuts_find_file.js.coffee

```diff
@@ -0,0 +1,15 @@
+#= require shortcuts_navigation
+
+class @ShortcutsFindFile extends ShortcutsNavigation
+  constructor: (@projectFindFile) ->
+    super()
+    _oldStopCallback = Mousetrap.stopCallback
+    Mousetrap.stopCallback = (event, element, combo) =>
+      if combo == 'up' or combo == 'down' or combo == 'esc' or combo == 'enter'
+        return element != @projectFindFile.inputElement[0]
+      return _oldStopCallback(event, element, combo)
```

Comment:
@DouweM 
When focus in textbox, Mousetrap's shortcuts is not working.
I think that we want to fire shortcuts action at that time in find file page.
so I overrided `Mousetrap.stopCallback`
and added `event.preventDefault` for avoiding to move cusor in textbox.
(when up/down key pressed, cusor is moved to home/end)